omc3/model/accelerators/lhc.py

```diff
@@ -200,21 +200,39 @@ def beam(self, value) -> None:
     def get_lhc_error_dir() -> Path:
         return LHC_DIR / "systematic_errors"
 
-    def get_variables(self, frm=None, to=None, classes=None):
+    def get_variables(self, frm: float = None, to: float = None, classes: Iterable[str] = None):
```

Comment:
I would personnally use `from_` but that's just a suggestion

Reply:
leaving it for now as this is not new in this PR and I don't want to deal with any consequences (but actually I don't think we are using it)